### ISSUE TITLE
define the narrative sections as being Markdown?

### ISSUE THREAD

**Issue by afeld** [opencontrol/schemas#13]
For the cloud.gov compliance, we are putting Markdown formatting in the `narrative` sections (maybe more)...should we say in the schema that Markdown is supported in that field?

**Comment by geramirez** [comment]
I think the markdown is a vestige to make the gitbook look nice. It should probably be taken out now that that schema supports text and keys in the narrative section. 

Thoughts?

**Comment by afeld** [comment]
https://github.com/18F/compliance-toolkit/issues/25 is around making the `narrative` more structured, but I'm still seeing things like links and ordered lists in those `text` sections. Maybe just sticking with plain text is ok for now. @jcscottiii Thoughts?

**Comment by jcscottiii** [comment]
@afeld yeah the idea is for now to stick to using plain text for now in order to get compatibility for both gitbook and docx. we might have to create some universal tags (sounds painful) that can be inserted and then transform it to the appropriate syntax for the specific output

**Comment by afeld** [comment]
Sounds good. Thanks!

**Comment by afeld** [comment]
Reopening, since the text is parsed by GitBook as Markdown.

**Comment by JJediny** [comment]
Would seem that full support for Markdown might be more then its worth but at the same time things like:

```
# Headings ? (maybe not as bold would me enough to sub for headings)
**bold**
*italics*
* bullets 
[link text](hyperlink)
[^1] references
`code`
```

are heavily used/relied on for creating sensible documentation via rendered markdown to html and/or in docx formats those would seem worth the time to investigate and inform the time/resources for each as a potential lift to support
